### DISCUSSION TITLE
Config lost each time

### DISCUSSION THREAD

**Discussion by toddlyons** [JulianPrieber/littlelink-custom#456] [Questions] [answered]
Hi,

After (manually) updating over the last few versions, I'm curious why configuration data is overwritten each time. 

For example: _Admin > Config > Footer links_

The sliders are back _on_ each time I install the update.zip, requiring me to turn _off_ all the link pages I don't want shown.

Thanks.

**Comment by toddlyons** [comment]
So... this was moved, suggesting it's not a bug so much as an intentional "feature"?

Sorry. I will refrain from filing further "bug" reports.

**Comment by JulianPrieber** [comment, marked as answer]
Config values are stored in the .env file in your root directory. You can just skip that file when updating.
Some updates require new options to be added to this file. If your system allows it, you can add these without resetting your config by visiting `example.com/update?finishing`

If that doesn't work for you, you may have to reset your config to get the new options.
